src/app.js

```diff
@@ -1,21 +1,56 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = 4200;
+const publicFolder = path.join(__dirname, 'public');
```

Comment:
Here, __dirname is `src`, while `public` is not a part of source code, it should be located on one level with `src`.